Backend/app/rider-platform/rider-app/Main/src/Beckn/ACL/OnUpdate.hs

```diff
@@ -318,12 +321,15 @@ parseRideStartedEvent order = do
   start <- Utils.getStartLocation stops & fromMaybeM (InvalidRequest "pickup stop is not present in RideStarted Event.")
   endOtp' <- start.stopAuthorization >>= (.authorizationToken) & fromMaybeM (InvalidRequest "authorization_token is not present in RideStarted Event.")
   let personTagsGroup = order.orderFulfillments >>= listToMaybe >>= (.fulfillmentAgent) >>= (.agentPerson) >>= (.personTags)
+  tagGroups <- order.orderFulfillments >>= listToMaybe >>= (.fulfillmentAgent) >>= (.agentPerson) >>= (.personTags) & fromMaybeM (InvalidRequest "fulfillment.agent.tags is not present in DriverArrived Event.")
```

Comment:
Don't throw error if tag is not there. Instead of that just consider odometer value as Nothing.